examples/tutorials/translations/português/Parte 11 - Classificação Segura Usando Aprendizagem Profunda.ipynb

```diff
@@ -0,0 +1,445 @@
+{
```

Comment:
<blockquote>If you enjoyed this and would like to join the movement toward privacy preserving, decentralized ownership of AI and the AI supply chain (data), you can do so in the following ways!</blockquote><blockquote>Se você gostou disso e gostaria de participar do movimento em prol da preservação da privacidade, da propriedade descentralizada da IA e da cadeia de fornecimento de IA (dados), você pode fazer isso das seguintes maneiras!</blockquote><blockquote>Se você gostou disso e gostaria de se juntar ao movimento em direção à preservação da privacidade, propriedade descentralizada da IA e da cadeia de suprimentos da AI (dados), você pode fazê-lo das seguintes maneiras!</blockquote><p> </p><p>I just realized it now, I think this is repeated and with a part in english</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2980/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
Done!

Reply:
Nice!